internal/datacoord/meta.go

```diff
@@ -363,16 +364,21 @@ func (m *meta) GetCollectionIndexFilesSize() uint64 {
 	m.RLock()
 	defer m.RUnlock()
 	var total uint64
+
+	missingCollections := make(typeutil.Set[int64])
 	for _, segmentIdx := range m.indexMeta.GetAllSegIndexes() {
 		coll, ok := m.collections[segmentIdx.CollectionID]
 		if ok {
 			metrics.DataCoordStoredIndexFilesSize.WithLabelValues(coll.DatabaseName,
 				fmt.Sprint(segmentIdx.CollectionID), fmt.Sprint(segmentIdx.SegmentID)).Set(float64(segmentIdx.IndexSize))
 			total += segmentIdx.IndexSize
 		} else {
-			log.Warn("not found database name", zap.Int64("collectionID", segmentIdx.CollectionID))
```

Comment:
There should not be a lot of logs since collections meta will reload asynchronously during the meta reload period in datacoord.

Reply:
these line shall be removed by other PRs, fyi